mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -253,30 +253,36 @@ CLASS com/mojang/blaze3d/systems/RenderSystem
 		ARG 2 width
 		ARG 3 height
 	CLASS class_5590 IndexBuffer
```

Comment:
```suggestion
	CLASS class_5590 ShapeElementBuffer
```
Maybe that's too verbose but I think IndexBuffer doesn't properly communicate that this holds/generates a pre-made element buffer for a specific shape. `PolygonElementBuffer` works too

Reply:
```suggestion
	CLASS class_5590 ShapeIndexBuffer
```
While it's an element buffer in OpenGL term, "element" is already used for `VertexFormatElement` (vertex attribute) in yarn. I think "index buffer" is fine for now.